site/zenodo_rdm/metrics/api.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""ZenodoRDM Metrics API."""
+
+import calendar
+from datetime import datetime, timedelta
+
+import requests
+from flask import current_app
+from invenio_accounts.models import User
+from invenio_communities.communities.records.models import CommunityMetadata
+from invenio_files_rest.models import FileInstance
+from invenio_search import current_search_client
+from invenio_search.utils import build_alias_name
+from opensearchpy import Search
+
+from zenodo_rdm.metrics.proxies import current_metrics
+
+
+class ZenodoMetric(object):
+    """API class for Zenodo Metrics."""
+
+    @staticmethod
+    def get_data_transfer():
+        """Get file transfer volume in TB."""
+        time_range = {"gte": current_metrics.metrics_start_date.isoformat()}
+
+        search = (
+            Search(
+                using=current_search_client,
+                index=build_alias_name("stats-file-download"),
+            )
+            .filter(
+                "range",
+                timestamp=time_range,
+            )
+            .params(request_timeout=120)
+        )
+        search.aggs.metric("download_volume", "sum", field="volume")
+        result = search[:0].execute().aggregations.to_dict()
+        download_volume = result.get("download_volume", {}).get("value", 0)
+
+        search = (
+            Search(using=current_search_client, index=build_alias_name("rdmrecords"))
```

Comment:
```suggestion
            Search(using=current_search_client, index=build_alias_name("rdmrecords-records"))
```

To ensure we query only published records (I think the top-level `rdmrecords` alias also includes draft indices).